.env.mainnet

```diff
@@ -0,0 +1,22 @@
+NEXT_PUBLIC_ENV=mainnet
+NEXT_PUBLIC_RIF_WALLET_SERVICES=https://rws.app.rootstockcollective.xyz
+NEXT_PUBLIC_EXPLORER=https://explorer.rootstock.io
+NEXT_PUBLIC_BUILD_ID=
+
+NEXT_PUBLIC_RIF_ADDRESS=0x2acc95758f8b5f583470ba265eb685a8f45fc9d5
+NEXT_PUBLIC_STRIF_ADDRESS=0x5db91e24bd32059584bbdb831a901f1199f3d459
+NEXT_PUBLIC_GOVERNOR_ADDRESS=0x71ac6ff904a17f50f2c07b693376ccc1c92627f0
+NEXT_PUBLIC_EA_NFT_ADDRESS=0x339f209b3eb8381c4fbe997232e95a21a731524c
+NEXT_PUBLIC_MULTICALL_ADDRESS=0xcA11bde05977b3631167028862bE2a173976CA11
+NEXT_PUBLIC_GRANTS_BUCKET_ADDRESS=0x48229e5D82a186Aa89a99212D2D59f5674aa5b6C
+NEXT_PUBLIC_GRANTS_ACTIVE_BUCKET_ADDRESS=0xf016fa6b237bb56e3aee7022c6947a6a103e3c47
+NEXT_PUBLIC_GROWTH_BUCKET_ADDRESS=0x267a6073637408b6A1d34d685ff5720A0CbCbD9d
+NEXT_PUBLIC_GENERAL_BUCKET_ADDRESS=0xfE3d9B7D68aE13455475F28089968336414FD358
+NEXT_PUBLIC_CHAIN_ID=30
+
+# BIM-related env variables
+NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x0000000000000000000000000000000000000000
+NEXT_PUBLIC_EPOCH_DURATION_IN_DAYS=0
+NEXT_PUBLIC_FIRST_EPOCH_START_DATE_ISO="1970-01-01T00:00:00Z"
+
+NEXT_PUBLIC_ENV_DATA_URL="https://raw.githubusercontent.com/rsksmart/bim-dao-frontend/develop/data.mainnet.json"
```

Comment:
404: Not Found

Reply:
Then for sure this needs to be removed.

Reply:
yea just done that. I miscommunicated something. I gave you the link as soon as I resolved conflicts, before making any of the necessary changes. I'm sorry for wasting your time like that. I should have made that clear.

Reply:
I still don't understand here. We can't merge this into the repo if the source of it is private. Is the plan to fix this after we merge this in?

Reply:
it will be found once it's merged to develop ;)